package.json

```diff
@@ -28,11 +28,17 @@
   },
   "homepage": "https://github.com/lob/hapi-bookshelf-total-count#readme",
   "devDependencies": {
```

Comment:
I don't think they are dependencies because they aren't required directly in the plugin. Instead Hapi consumes the register function, and Bookshelf models are injected in the options of the plugin.